ProcessQueue/__init__.py

```diff
@@ -476,3 +479,71 @@ def main(msg: func.QueueMessage):
             field_ids = []
             field_names = []
             data = []
+    if parent_sr_id is not "None":

```

Comment:
meant to be None, rather than "None" ?

Reply:
ahh no, it comes from some trick you do later with `azure_dict['parent_SR_id'] = "None"`